app/(dashboard)/mailboxes/[mailbox_slug]/settings/chat/components/WidgetInstallation.tsx

```
@@ -0,0 +1,420 @@
+"use client";
+
+import { ExternalLink, Sparkles } from "lucide-react";
+import { useState } from "react";
+import { getBaseUrl, getMarketingSiteUrl } from "@/components/constants";
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "@/components/ui/accordion";
+import { Button } from "@/components/ui/button";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
+import { RouterOutputs } from "@/trpc";
+import CodeBlock from "../codeBlock";
+import SectionWrapper from "../../sectionWrapper";
```

ESLint

[error] Replace `CodeBlock·from·"../codeBlock";⏎import·SectionWrapper·from·"../../sectionWrapper` with `SectionWrapper·from·"../../sectionWrapper";⏎import·CodeBlock·from·"../codeBlock`

(prettier/prettier)